packages/react-native/package.json

```diff
@@ -97,6 +97,7 @@
     "@react-native-community/cli": "12.0.0-alpha.15",
     "@react-native-community/cli-platform-android": "12.0.0-alpha.15",
     "@react-native-community/cli-platform-ios": "12.0.0-alpha.15",
+    "@react-native-community/cli-tools": "12.0.0-alpha.15",
```

Comment:
Mentioned offline, could we not add a new dependency (and diff) and instead refactor so we don't need to take a direct dependency here?